src/components/JUPTable/JUPTable.tsx

```diff
@@ -183,7 +185,7 @@ const JUPTable: React.FC<IJUPTableProps> = ({ headCells, rows, title, path, Disp
   return (
     <TableBackground>
       <TableContainer>
-        <TableTitle title={title} path={path} DisplayedComponents={DisplayedComponents} />
+        <TableTitle title={title ? title : undefined} path={path} DisplayedComponents={DisplayedComponents} />
```

Comment:
this change should be unnecessary as the TableTitle interface marks the title as optional

Reply:
fixed